### ISSUE TITLE
[APP SUBMITTED]: SHOWQUEUE-FORCE-UPDATE :: [2404f19] No data returned from theTVDB, unable to update this show.

### ISSUE THREAD

**Issue by StanESmith** [SickChill/SickChill#5734]
### INFO
Python Version: **2.7.16 (default, Oct 10 2019, 22:02:15) [GCC 8.3.0]**
Operating System: **Linux-5.0.21-5-pve-x86_64-with-debian-10.2**
Locale: UTF-8
Branch: **master**
Commit: SickChill/SickChill@2404f197697602a2bbfa8e6b39207229a94d37bb
Link to Log: https://gist.github.com/4e1324af1ed0f75f9d5c4335eb504520
### ERROR
```
SHOWQUEUE-FORCE-UPDATE :: [2404f19] No data returned from theTVDB, unable to update this show.
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by tstormgit** [comment]
### INFO
Python Version: **2.7.16 (default, May 16 2019, 01:14:27) [GCC 4.2.1 Compatible FreeBSD Clang 6.0.0 (tags/RELEASE_600/final 326565)]**
Operating System: **FreeBSD-11.2-STABLE-amd64-64bit-ELF**
Locale: UTF-8
Branch: **master**
Commit: SickChill/SickChill@2404f197697602a2bbfa8e6b39207229a94d37bb
Link to Log: https://gist.github.com/5be9b6ee28f51fe8a23baccb496eae5f
### ERROR
```
SHOWQUEUE-FORCE-UPDATE :: [2404f19] No data returned from theTVDB, unable to update this show.
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by augur42** [comment]
I too had this issue just in the last hour, traced it to a particular show (the blacklist) and it had no episodes (it had lost them all), clicked on Force Full Update a few times with it failing each time and eventually it repopulated with all episodes. Looks like this is a temporary issue on tvdb end.

**Comment by tstormgit** [comment]
That worked for me, I wasn't getting any Sick Chill errors today but it didn't populate with any of the new info, did your trick of trying it a few times and eventually did it's thing, thanks augur42.

**Comment by jclemens78** [comment]
### INFO
Python Version: **2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-10-10.0.17763**
Locale: cp1252
Branch: **master**
Commit: SickChill/SickChill@2404f197697602a2bbfa8e6b39207229a94d37bb
No Log available with ERRORS:
### ERROR
```
SHOWQUEUE-FORCE-UPDATE :: [2404f19] No data returned from theTVDB, unable to update this show.
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by miigotu** [comment]
Fixed in #5789 (Live on develop, needs tests)
